tests/e2e/specs/LoadComplexData.spec.js

```diff
@@ -34,13 +31,10 @@ describe('Validation Rules (Advanced test)', () => {
       'FPP_SUB_TOTAL_FACULTY_TL_FULL_TIME': '0',
       'FPP_SUB_TOTAL_PREMIUM_TL_FULL_TIME': '0',
       'FPP_REQUEST_ID': '',
-      // 'FPP_USER_PART_TIME': {
-      //   'FPP_TL_PART_TIME': null,
-      //   'FPP_NTL_PART_TIME': null,
-      // },
-      // @todo this was overwritten to null by SelectList when list is refreshed
-      // at FormSelectList.vue @ updateWatcherDependentFieldValue()
-      'FPP_USER_PART_TIME': null,
+      'FPP_USER_PART_TIME': {
```

Comment:
This should not be changed, because it was fixed by a previous PR (not yet merged): https://github.com/ProcessMaker/screen-builder/pull/1303